DROD/GameScreen.cpp

```diff
@@ -3020,7 +3020,7 @@ int CGameScreen::HandleEventsForPlayerDeath(CCueEvents &CueEvents)
 		//Fade to black.
 		if (g_pTheBM->bAlpha && dwStart)
 		{
-			const float durationFraction = min(1, (dwNow - dwStart) / (float)dwDeathDuration);
+			const float durationFraction = min((float)1, (dwNow - dwStart) / (float)dwDeathDuration);
```

Comment:
Wouldn't `1.0f` be preferable?

Reply:
Yep, let's do it as 1.0f, as the standard idiom for typing a numeric value in C/C++

Reply:
Sure!